templates/index.json-ld

```diff
@@ -110,28 +110,6 @@
 				"path": "theme-fr.html"
 			},
 
-			{
-				"title": "[Regulation name] <small>([NN-XX])</small>",
```

Comment:
@duboisp Shouldn't it be in the master list also?

Reply:
No, only in the template. This main json-ld is only a temporary measure to take the time of creating every single one.